documentation/release_notes.markdown

```diff
@@ -16,6 +16,22 @@ canonical: "/puppetdb/latest/release_notes.html"
 [stockpile]: https://github.com/puppetlabs/stockpile
 [queue_support_guide]: ./pdb_support_guide.html#message-queue
 
+5.1.2
+-----
+
+PuppetDB 5.1.2 is a bugfix and performance release.
+
+### Bug Fixes
+* This release containes a fix for a uniqueness bug in the recent fact data migration.
```

Comment:
I'd leave off "This release", and also maybe match the other entry, i.e. "Fixed a ..." to match "Added a ...".  Though I might go even further and write both in NEWS style, i.e. "A recent fact data migration should no longer crash when the existing data has unexpected null value representations."